packages/spruce-next-helpers/src/skillskit/index.js

```diff
@@ -106,6 +30,9 @@ const skill = {
 	},
 
 	ready: function({ resetUrlTrail = false, showHeader = true } = {}) {
+		Iframes.sendMessage({
```

Comment:
🤔 did you mean to leave this here? Not sending anything.

Reply:
oops, yeah, did not mean to leave that there!